CSVParser/ParsersOVerview.drawio.svg

```diff
@@ -0,0 +1,327 @@
+                        <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
+                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
+                                <div style="color: rgb(36 , 41 , 46) ; background-color: rgb(255 , 255 , 255) ; font-family: &quot;menlo&quot; , &quot;monaco&quot; , &quot;courier new&quot; , monospace ; line-height: 18px">
+                                    field
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="70" y="209" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    field
+                </text>
+            </switch>
+        </g>
+        <path d="M 80 130 L 115.5 94.5" fill="none" stroke="#cccccc" stroke-miterlimit="10" pointer-events="stroke"/>
+        <path d="M 119.21 90.79 L 116.73 98.22 L 115.5 94.5 L 111.78 93.27 Z" fill="#cccccc" stroke="#cccccc" stroke-miterlimit="10" pointer-events="all"/>
+        <rect x="50" y="130" width="60" height="30" fill="rgb(255, 255, 255)" stroke="#cccccc" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 58px; height: 1px; padding-top: 145px; margin-left: 51px;">
+                        <div data-drawio-colors="color: #CCCCCC; " style="box-sizing: border-box; font-size: 0px; text-align: center;">
+                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(204, 204, 204); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
+                                <div style="background-color: rgb(255 , 255 , 255) ; font-family: &quot;menlo&quot; , &quot;monaco&quot; , &quot;courier new&quot; , monospace ; line-height: 18px">
+                                    name
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="80" y="149" fill="#CCCCCC" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    name
+                </text>
+            </switch>
+        </g>
+        <path d="M 220 60 L 220 36.37" fill="none" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="stroke"/>
```

Comment:
I lighter color than black would look better for those that use a dark them on github.

Reply:
Due to GitHub limitations, it's impossible to create a picture that looks perfect in both light and dark scheme. But it should look better now.